server/characters/characters_common.py

```diff
@@ -29,4 +32,27 @@ def db_connection():
     if config.has_option('Characters','characters_psql_pass'):
         db_pass = config.get('Characters','characters_psql_pass')
     connection = psycopg2.connect("dbname=%s user=%s password=%s" % (db, username,db_pass))
-    return connection
\ No newline at end of file
+    return connection
+
+def fetchall_from_db_query(query):
+    """Run a query on the character database and return all of the results."""
+    connection = db_connection()
+    myCursor = connection.cursor()
+    myCursor.execute(query)
+    returnMe = myCursor.fetchall()
+    myCursor.close()
+    connection.commit()
+    return returnMe
+
+def fetch_first_from_db_query(query):
+    """Run a query on the character database and return the first of the results"""
+    all_rows = fetchall_from_db_query(query)
```

Comment:
Is there no more efficient way to grab the first record than asking for all of them and dropping numbers 2 thru infinity?

Reply:
Since we're querying on the primary key, there shouldn't _be_ a numbers 2 through infinity. I could use the cursor's fetchone() method, and write a similar wrapper around that as well, but then we'd have more code to keep up. On the other hand, code is meant to be read, and fetchall/grab the first seems to be less intuitive than fetchone. I'll change it.